tests/layer1/test_layer1_speed_Novus_100G.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+from abstract_open_traffic_generator.port import *
+from abstract_open_traffic_generator.config import *
+from abstract_open_traffic_generator.layer1 import *
+from abstract_open_traffic_generator.control import *
+
+
```

Comment:
we should ensure that this test doesn't run unless we provide a specific tag. e.g. if "e2e" or "not e2e" tags are provided, do not run this test.

Reply:
didn't get the comment. "e2e" tag is already added, are you expecting to add a reason?